android/assets/jsons/translationsByLanguage/Italian.properties

```diff
@@ -481,6 +480,10 @@ Songhai = Songhai
 Askia = Askia
 Receive triple Gold from Barbarian encampments and pillaging Cities. Embarked units can defend themselves. = Saccheggiare città o conquistare accampamenti barbari frutta una quantità tripla di Oro. Le unità imbarcate possono difendersi da sole.
 
+Spain = Spagna
+Isabella = Isabella
+Gold bonus for discovering a Natural Wonder (bonus enhancef if first to discover it). Culture, Happiness and tile yelds from Natural Wonders. = Otteni Oro bonus (raddoppiato se la scopri per prima) quando scopri una Meraviglia Naturale, che donano il doppio di Cultura, Felicità e rese di casella.
```

Comment:
There are some typos in the non translated string. Also I cannot find these lines in the template.properties, did you add them by hand?